airbyte-server/src/main/java/io/airbyte/server/apis/ConfigurationApi.java

```diff
@@ -552,12 +552,9 @@ public ImportRead importArchive(@Valid File archiveFile) {
     try {
       return call.call();
     } catch (ConfigNotFoundException e) {
-      throw new KnownException(
-          HttpStatus.UNPROCESSABLE_ENTITY_422,
-          String.format("Could not find configuration for %s: %s.", e.getType().toString(), e.getConfigId()), e);
+      throw new IdNotFoundException(String.format("Could not find configuration for %s: %s.", e.getType().toString(), e.getConfigId()), e);
```

Comment:
seeing these various exceptions that extend `KnownException` next to each other, I'm wondering if there's any naming convention we should adopt so it's clearer what these exceptions are doing. Because they're now not obviously known exceptions its hard to get the queue that these will be handled beautifully in the API. e.g. add KnownException (or some abbreviation, or some other naming convention entirely).

Reply:
Yeah, I agree.  They should all include KnownException in the name, so that the hierarchy is more obvious, and also so they show up as autocomplete suggestions during IDE searches.